<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_settings</class>
 <widget class="QDialog" name="Dialog_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>304</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>479</width>
    <height>304</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>479</width>
    <height>304</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/filmonaizer.qrc">
    <normaloff>:/icons/gear.png</normaloff>:/icons/gear.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,5">
     <item>
      <widget class="QGroupBox" name="groupBox_other">
       <property name="title">
        <string>Other</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_saving_mode_traffic">
          <property name="text">
           <string>Saving mode traffic</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_clear_temp">
          <property name="text">
           <string>Clear directory Temp a exit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>171</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_proxy">
       <property name="title">
        <string>Proxy</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_enable_proxy">
          <property name="text">
           <string>Enable proxy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_proxy"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,10">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Address proxy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ip">
          <property name="toolTip">
           <string>ip</string>
          </property>
          <property name="placeholderText">
           <string>ip</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_port">
          <property name="toolTip">
           <string>port</string>
          </property>
          <property name="placeholderText">
           <string>port</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,10">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Authentication</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_username">
          <property name="toolTip">
           <string>username</string>
          </property>
          <property name="placeholderText">
           <string>username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="toolTip">
           <string>password</string>
          </property>
          <property name="placeholderText">
           <string>password</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_add_proxy">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Add proxy</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons/filmonaizer.qrc">
              <normaloff>:/icons/new/plus-button.png</normaloff>:/icons/new/plus-button.png</iconset>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_delete_proxy">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Remove proxy</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons/filmonaizer.qrc">
              <normaloff>:/icons/new/minus-button.png</normaloff>:/icons/new/minus-button.png</iconset>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/filmonaizer.qrc"/>
 </resources>
 <connections/>
</ui>
